src/sentry-global-search/scss/styles.scss

```diff
@@ -5,7 +5,7 @@
 
   --sgs-color-progress-indicator: black;
 
-  --sgs-color-result-heading-backgorund: black;
+  --sgs-color-result-heading-background: black;
```

Comment:
This appears to be a local variable which is otherwise harmless. But using misspelled items is an invitation to errors in the future.